### PR TITLE
Theme Builder : UI - New - Dialog Component.

### PR DESCRIPTION
**Pull request by Nevoss**

### CODE REVIEW

File: core/app/modules/site-editor/assets/js/ui/dialog/dialog.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import Button from 'elementor-app/ui/molecules/button';
+import './dialog.scss';
+
+export default function Dialog( props ) {
+	const WrapperTag = props.wrapperTag;
+	const wrapperProps = {
+		...'form' === props.wrapperTag && { onSubmit: props.onSubmit },
```
Comment (matipojo) [inline core/app/modules/site-editor/assets/js/ui/dialog/dialog.js:7]: Simplify.

**Review by matipojo — CHANGES_REQUESTED** [review]

File: core/app/modules/site-editor/assets/js/organisms/error-dialog.js [hunk 2]
```diff
@@ -8,34 +8,38 @@ import DialogContent from '../ui/dialog/dialog-content';
...
 
 ErrorDialog.propTypes = {
 	title: PropTypes.string,
 	text: PropTypes.string,
-	learnMoreUrl: PropTypes.string,
-	goBackHandler: PropTypes.func,
+	learnButtonUrl: PropTypes.string,
+	backButtonHandler: PropTypes.func,
+	learnButtonText: PropTypes.string,
+	backButtonText: PropTypes.string,
```
Comment (matipojo) [inline core/app/modules/site-editor/assets/js/organisms/error-dialog.js:35]: Reorder here and on defaultProps.

```suggestion
	learnButtonText: PropTypes.string,
	learnButtonUrl: PropTypes.string,
	backButtonText: PropTypes.string,
	backButtonHandler: PropTypes.func,
```

**Review by matipojo — CHANGES_REQUESTED** [review]

**Review by matipojo — DISMISSED** [review]

**Review by joshmarom — CHANGES_REQUESTED** [review]

**Review by joshmarom — DISMISSED** [review]

File: core/app/assets/js/ui/molecules/buttons-api.scss [hunk 3]
```diff
@@ -73,7 +81,12 @@ $button-dark-disabled-color: dark-theme-colors(light);
   --button-secondary-hover-background-color: #{$button-secondary-hover-background-color};
   --button-secondary-active-background-color: #{$button-secondary-active-background-color};
   --button-secondary-color: #{$button-secondary-color};
-  // cta
+	// danger
+	--button-danger-background-color: #{$button-danger-background-color};
+	--button-danger-hover-background-color: #{$button-danger-hover-background-color};
+	--button-danger-active-background-color: #{$button-danger-active-background-color};
+	--button-danger-color: #{$button-danger-color};
+	// cta
```
Comment (kobizz) [inline core/app/assets/js/ui/molecules/buttons-api.scss:89]: It looks like a wrong indentation

File: core/app/assets/js/ui/dialog/dialog.scss [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+		display: flex;
+	}
+
+	&__button {
+		flex: 1;
+		border-top: 1px solid var(--hr-color);
+		line-height: spacing(44);
+		text-align: center;
+		justify-content: center;
+		&:last-child:not(:first-child) {
```
Comment (kobizz) [inline core/app/assets/js/ui/dialog/dialog.scss:35]: Insert an empty line between rules and an internal selector

**Review by kobizz — CHANGES_REQUESTED** [review]

**Review by kobizz — APPROVED** [review]

**Review by matipojo — APPROVED** [review]

**Pull request merged**